code/src/reset-style.css

```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
Cool that you did a reset to the style! Did it make the styling easier? Very curious!